docs/tutorial/code/mygc_semispace/mutator.rs

```diff
@@ -69,14 +76,11 @@ pub fn create_mygc_mutator<VM: VMBinding>(
         allocator_mapping: &*ALLOCATOR_MAPPING,
         // Modify
         // ANCHOR: space_mapping
-        space_mapping: box vec![
-            (AllocatorSelector::BumpPointer(0), mygc.tospace()),
-            (
-                AllocatorSelector::BumpPointer(1),
-                mygc.common.get_immortal(),
-            ),
-            (AllocatorSelector::LargeObject(0), mygc.common.get_los()),
-        ],
+        space_mapping: box {
```

Comment:
The tutorial needs to be updated for the changes. For example, [here](https://github.com/mmtk/mmtk-core/blob/4bd6ca82ad037dac97920bf3a1408e4912a6bf04/docs/tutorial/src/mygc/ss/alloc.md?plain=1#L215), we no longer set space mapping for LOS and immortal in the plan. Can you please check if there is other outdated text in the tutorial?

Reply:
I updated the text. Could you please take a read again? Thanks!